### PR TITLE
Add logic to recognize the path of the "common" files.

### PR DESCRIPTION
**Pull request by jansiegel**
### Context
This PR fixes the problem described in #9534.

As discussed, the logic targets only the "frameworked" docs versions.

### How has this been tested?
Run the docs using:
- `docs:start:no-cache`
- `docs:build:no-cache`

For setups:
- With just `next` as the framework version
- The latest version as frameworked
- Version before latest as frameworked

And running the examples both in the docs and JSFiddle. (In JSFiddle mostly checking the file URLs, as they're not present on production yet)

[skip changelog]

### Related issue(s):
1. #9534 

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Review by wszymanski — COMMENTED** [review]
> For setups:
> 
> * With just `next` as the framework version
> * The latest version as frameworked
> * Version before latest as frameworked

The above criteria are met. However, I have some things for discussion. Thus, I'm leaving PR yet.

**Comment by wszymanski** [comment]
PS. I can see lot of warnings:

```
Can't find the "@/api/metaSchema.md" permalink. Does this file exist?
Can't find the "@/api/pluginHooks.md" permalink. Does this file exist?
```

I'll check whether it's related to the branch.

**Comment by wszymanski** [comment]
I don't know whether it is related to the branch, but it seems that:

- images aren't shown
- link in the framework drop-down menu doesn't work
- links doesn't work (for example, see `Guides for your framework` section below).

Screenshot 2022-06-10 at 11 38 28

File: docs/.vuepress/theme/components/ThemeSwitcher.vue [hunk 1]
```diff
@@ -136,14 +140,15 @@ export default {
...
-  bottom: 4px;
   top: 0;
   bottom: 0;
   margin: auto 0;
   -webkit-transition: 0.4s;
   transition: 0.4s;
   box-shadow: 0 0px 3px #2020203d;
-  background: #ffffff url('/docs/img/light-theme-icon.svg');
+  /* Fallback for IE, should work in production */
+  background: #ffffff url('/docs/javascript-data-grid/img/light-theme-icon.svg');
```
[the remark this answers is not in the document]
Comment (jansiegel) [reply docs/.vuepress/theme/components/ThemeSwitcher.vue:150]: > I'm not even sure whether using some VuePress variable will be possible in Stylus' CSS - probably no
Unfortunately, it's not possible with Vue 2 (it is in Vue 3.2+)

I'm not a huge fan of hardcoding it as well, but it's the best approach I could think of. It's only for IE, though, as [all the modern browsers support the CSS variables](https://caniuse.com/css-variables), so this line would be ignored 99% of the time.

**Review by wszymanski — APPROVED** [review]
Good job 👌

**Pull request merged**